weave/trace_server/migrations/002_feedback.up.sql

```diff
@@ -0,0 +1,49 @@
+
+CREATE TABLE feedback (
+    /*
+    `id`: The unique identifier for the feedback. This is a UUID.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier for the call. This is an internal
+    identifier that matches the project identifier in the W&B API.
+    It is stored for feedback to allow efficient permissions filtering.
+    */
+    project_id String,
```

Comment:
The intention is that the project id would match the call's project id. This would be an issue to handle if we ever allow things like moving calls between projects.

We could also not store it, and instead query the calls table to get the information we need to do permissions checks, but my unconfirmed suspicion was that it would be more complicated and inefficient to do it that way.

Reply:
In that case, we may want to rename this to `call_project_id`. Otherwise, this feels like it should be the project that `weave.init` was called with during the session when the user is submitting feedback which may not necessarily match.

Reply:
Leaving name as `project_id` since in new version the feedback might be on something other than a call.